<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ransacDialog</class>
 <widget class="QDialog" name="ransacDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RANSAC Settings</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::No|QDialogButtonBox::Yes</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>RANSAC FIT</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="maxNum">
   <property name="geometry">
    <rect>
     <x>264</x>
     <y>120</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>5000</number>
   </property>
   <property name="singleStep">
    <number>20</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="thresholdId">
   <property name="geometry">
    <rect>
     <x>265</x>
     <y>160</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
   <property name="decimals">
    <number>5</number>
   </property>
   <property name="singleStep">
    <double>0.000010000000000</double>
   </property>
   <property name="value">
    <double>0.000100000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="randomPercent">
   <property name="geometry">
    <rect>
     <x>266</x>
     <y>200</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="value">
    <double>15.000000000000000</double>
   </property>
  </widget>
  <widget class="QSpinBox" name="Iterations">
   <property name="geometry">
    <rect>
     <x>264</x>
     <y>80</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>100000</number>
   </property>
   <property name="singleStep">
    <number>20</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="finalPercent">
   <property name="geometry">
    <rect>
     <x>267</x>
     <y>240</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="value">
    <double>40.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>85</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Max Iterations</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>124</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Max number of random sample</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>44</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Best error : </string>
   </property>
  </widget>
  <widget class="QLabel" name="error">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>44</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Error</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>166</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ideal Threshold</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>205</y>
     <width>201</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Random data percentage</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>246</y>
     <width>221</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Min &quot;nice&quot; data percentage</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>280</y>
     <width>261</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Retry using the parameters above?</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ransacDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ransacDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
